service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -66,13 +66,13 @@
     private UBSManagementService ubsManagementService;
     private final LanguageRepository languageRepository;
     private final CourierLocationRepository courierLocationRepository;
-    @Value("${fondy.payment.key}")
+    @Value("$ {greencity.fondy-payment-key}")
     private String fondyPaymentKey;
-    @Value("${merchant.id}")
+    @Value("$ {greencity.merchant-id}")
     private String merchantId;
-    @Value("${liqpay.public.key}")
+    @Value("${greencity.liq-pay-public-key}")
     private String publicKey;
-    @Value("${liqpay.private.key}")
+    @Value("${greencity.liq-pay-private-key}")
```

Comment:
why do you inject @ Values here? You've created GreenCityConfigurationProperties , so use it everywhere.